docroot/themes/humsci/humsci_basic/templates/menus/pager.html.twig

```diff
@@ -64,7 +64,7 @@
           {% else %}
             {% set title = 'Go to page @key'|t({'@key': key}) %}
           {% endif %}
-          <a href="{{ item.href }}" title="{{ title }}"{{ item.attributes|without('href', 'title') }}>
+          <a href="{{ item.href }}" title="{{ title }}"{{ item.attributes|without('href', 'title').addClass(current == key ? 'pager__link' : '') }}>
```

Comment:
Is there a reason for not adding this class to all links? Semantically, it doesn't make sense to have it only in the active items.